### PR TITLE
Added home page

### PR DESCRIPTION
**Pull request by andrewdempsey2018**
Added home page:

* Desktop
* 
image

* Mobile
* 
image

### CODE REVIEW

File: templates/home.html [hunk 1]
```diff
@@ -1,5 +1,92 @@
...
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="column">
+            <div class="card card-mission-4">
+                <div class="card-content">
+                    <div class="content">
+                        "When efvery person in the world is treated as equal, no matter who they are or whom they love,
```
Comment (auxfuse) [inline templates/home.html]: Typo in "every".

File: static/css/style.css [hunk 2]
```diff
@@ -72,47 +72,83 @@ body {
...
+            var(--blue), var(--blue),
+            var(--purple), var(--purple));
     -webkit-background-clip: text;
     -webkit-text-fill-color: transparent;
     cursor: pointer;
     transition: all .6s cubic-bezier(0.86, 0, 0.07, 1);
 }
 
-#chevron-checkbox:checked ~ .chevron-checkbox-label>.fa-heart {
+#chevron-checkbox:checked~.chevron-checkbox-label>.fa-heart {
```
Comment (auxfuse) [inline static/css/style.css]: Can this be returned to the version that it is above? With the spaces around the `~`.

File: static/css/style.css [hunk 3]
```diff
@@ -72,47 +72,83 @@ body {
...
-        var(--green), var(--green),
-        var(--blue), var(--blue),
-        var(--purple), var(--purple)
-    );
+            var(--red), var(--red),
+            var(--orange), var(--orange),
+            var(--yellow), var(--yellow),
+            var(--green), var(--green),
+            var(--blue), var(--blue),
+            var(--purple), var(--purple));
```
Comment (auxfuse) [inline static/css/style.css]: Can the last `)` be on it's own line? Much like it is above in the red?

**Review by auxfuse — CHANGES_REQUESTED** [review]
Hi Andrew, requested a few changes? :) Can you resolve conflicts also and then push to the branch please?

**Review by lemocla — APPROVED** [review]
LGTM 🎉

**Review by auxfuse — APPROVED** [review]
LGTM

**Pull request merged**